coordinator/src/services/translation/prescription/organization.ts

```diff
@@ -35,24 +31,30 @@ export function convertOrganizationAndProviderLicense(
 function convertRepresentedOrganization(
   fhirOrganization: fhir.Organization,
   fhirHealthcareService: fhir.HealthcareService,
-  fhirBundle: fhir.Bundle,
-  isCancellation: boolean
+  fhirBundle: fhir.Bundle
 ) {
+  const representedOrganization = isNhsTrust(fhirOrganization)
+    ? new CostCentreHealthcareService(fhirHealthcareService)
+    : new CostCentreOrganization(fhirOrganization)
+  return convertRepresentedOrganizationDetails(representedOrganization, fhirBundle)
+}
+
+function isNhsTrust(fhirOrganization: fhir.Organization) {
   const organizationTypeCoding = getCodeableConceptCodingForSystemOrNull(
-    fhirOrganization.type,
+    fhirOrganization.type || [],
```

Comment:
It would be nicer if the handling for an invalid organizationType could be handled inside the `getCodeableConceptCodingForSystemOrNull` function. Then we don't need to leak out the logic into the calling function.

Reply:
Yeah that's fair I was being lazy